### PR TITLE
Pill improvements

### PR DESCRIPTION
**Pull request by keirog**
Changes to allow the pills to be `<buttons>` as well as `<a>`s. Some minor tidy-ups included. There’s an existing issue with the disabled state but I’ll fix that in a separate PR.

No visual differences to show 😊

### CODE REVIEW

File: components/concept-pill/main.scss [hunk 1]
```diff
@@ -17,37 +20,47 @@
 
 .n-myft-concept-pill {
 	@include oTypographySansBold($scale: 0);
+	background-color: transparent;
 	border: 1px solid getColor('claret');
 	border-radius: 100px;
+	box-sizing: content-box;
```
Comment (debugwand) [inline components/concept-pill/main.scss]: this is default, is it overwriting something?

File: components/concept-pill/main.scss [hunk 2]
```diff
@@ -17,37 +20,47 @@
...
 }
 
 .n-myft-concept-pill:hover {
 	border: 1px solid getColor('claret');
+}
 
-	&.n-myft-concept-pill--add::before {
-		@include plus-icon('black-70');
-	}
+.n-myft-concept-pill--added:hover,
```
Comment (debugwand) [inline components/concept-pill/main.scss]: should these also have `:focus` for keyboard users?

File: components/concept-pill/main.scss [hunk 3]
```diff
@@ -58,25 +71,33 @@
 }
 
 .n-myft-concept-pill--disabled,
-.n-myft-concept-pill--disabled:hover {
-	border: 1px solid getColor('black-70');
-	color: getColor('black-70');
+.n-myft-concept-pill[disabled],
+.n-myft-concept-pill[disabled].n-myft-concept-pill--inverse {
+	&,
+	&:hover {
```
Comment (debugwand) [inline components/concept-pill/main.scss]: same as above for `:focus`

File: components/concept-pill/main.scss (hunk 1 above)
Comment (keirog) [reply components/concept-pill/main.scss]: ah it’s to make sure the same is applied for `<button>` too

File: components/concept-pill/main.scss (hunk 2 above)
Comment (keirog) [reply components/concept-pill/main.scss]: oof definitely, thanks 😳

Comment (keirog) [reply components/concept-pill/main.scss]: just realised the omission was because we have a separate focus style:
focus

**Pull request merged**